### DISCUSSION TITLE
Issues with srt2subtitles install

### DISCUSSION THREAD

**Discussion by leoruby33** [CommandPost/FCPCafe#268] [Final Cut Pro Questions] [answered]
I'm having some trouble installing srt2subtitles--installation of Node and the tool seem to go fine, but on running I get this output: 

`When attempting to run srt2subtitles, I get the follow error:

node:internal/modules/esm/resolve:844
  throw new ERR_MODULE_NOT_FOUND(packageName, fileURLToPath(base), null);
        ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'xmlbuilder2' imported from /opt/homebrew/lib/node_modules/srt2subtitles/utils/fcpxml.js
    at packageResolve (node:internal/modules/esm/resolve:844:9)
    at moduleResolve (node:internal/modules/esm/resolve:901:20)
    at defaultResolve (node:internal/modules/esm/resolve:1121:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:396:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:365:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:240:38)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)
    at link (node:internal/modules/esm/module_job:84:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}
`

Any idea what I'm doing wrong here?

**Comment by latenitefilms** [comment]
I'm not sure to be honest. Maybe @shaishaicookie can jump in?

Did you install via `npm install --global srt2subtitles`?

Does entering `srt2subtitles --help` correctly display the help?

Are you supplying the frame rate? For example,  `srt2subtitles /path/to/file.srt 25`?

**Comment by leoruby33** [comment]
@latenitefilms thanks Chris. I installed via `npm install --global srt2subtitles`; I get the output I posted above after running `srt2subtitles --help`. So far I have not gone beyond entering this, so I haven't even gotten to entering the framerate or using the program.

**Comment by shaishaicookie** [comment, marked as answer]
Thanks for reaching out! I have already resolved the issue. The problem was caused by my previous oversight of not including 'xmlbuilder2' in the dependencies. Now, everything is working perfectly.

I have just retested it. Additionally, this repository includes an SRT file that you can use for testing.

Sincerely,
Sherry

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: leoruby33 ***@***.***>
发送时间: Monday, October 30, 2023 11:45:10 PM
收件人: CommandPost/FCPCafe ***@***.***>
抄送: Sherry Wang ***@***.***>; Mention ***@***.***>
主题: Re: [CommandPost/FCPCafe] Issues with srt2subtitles install (Discussion #268)


@latenitefilms<https://github.com/latenitefilms> thanks Chris. I installed via npm install --global srt2subtitles; I get the output I posted above after running srt2subtitles --help. So far I have not gone beyond entering this, so I haven't even gotten to entering the framerate or using the program.

―

**Reply by leoruby33** [reply]
Thank you!
